### ISSUE TITLE
Fatal error in libGL.so.1 during startup (SIGSEGV in libGL.so)

### ISSUE THREAD

**Issue by execthis** [jsettlers/settlers-remake#274]
Hey mates

There seems to be a problem running JSettlers on Linux with OpenJDK 8 and in my case the depending libGL from nvidia. I was just setting up anything to start my personal duty  / contribution for this awesome project! (Y) :) Same result with OpenJDK 7 as excpectedj bc of GL library. Is someone having the same trouble during startup?

**System:**
Arch Linux with GDM 3.18.2
Linux ---- 4.3.3-2-ARCH #1 SMP PREEMPT Wed Dec 23 20:09:18 CET 2015 x86_64 GNU/Linux
nvidia version 358.16 
xorg version 1.18.0

**Exact error output during startup from terminal "java -jar JSettlers.jar":**

```
#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007ff797b1f9bd, pid=18177, tid=140701586265856
#
# JRE version: OpenJDK Runtime Environment (8.0_66-b17) (build 1.8.0_66-b17)
# Java VM: OpenJDK 64-Bit Server VM (25.66-b17 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# C  [libGL.so.1+0x539bd]
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
# The crash happened outside the Java Virtual Machine in native code.
# See problematic frame for where to report the bug.
#
```

Please find thread and memory dump attached  Tell me if you need more infos.

regards

Edit: Im going to provide more logs later on!

[hs_err_pid16691.txt](https://github.com/jsettlers/settlers-remake/files/83876/hs_err_pid16691.txt)

**Comment by andreas-eberle** [comment]
@michaelzangl: Can you investigate this further? Could it be an issue with JOGL?

**Comment by michaelzangl** [comment]
It is pretty hard to debug. It seems to be a jogl issue since JOGL checks the arguments passed on to the graphics driver.

@execthis Can you enable core dumping and upload the core dump? We might then be able to work around this issue.

**Comment by execthis** [comment]
@michaelzangl I will upload the core dump. Its something between JOGL and the nvidia based libGL. Im pretty sure that it would work perfectly with mesa libGL.

**Comment by Fantu** [comment]
I have the same error on gaming pc with nvidia proprietary driver, instead on notebook with intel opensource driver works

**Comment by adrianer** [comment]
The problem seems not limited to the OpenJDK - happens with with Oracle's Java too:

Java(TM) SE Runtime Environment (build 1.8.0_72-b15)
Java HotSpot(TM) 64-Bit Server VM (build 25.72-b15, mixed mode)

NVIDIA driver 361.28 on Kubuntu 14.04.

**Comment by ckorn** [comment]
Also happens here on Ubuntu 15.10 amd64
NVIDIA 355.11 drivers (GeForce GTX 750 Ti)
```
openjdk version "1.8.0_66-internal"
OpenJDK Runtime Environment (build 1.8.0_66-internal-b17)
OpenJDK 64-Bit Server VM (build 25.66-b17, mixed mode)
```


http://sprunge.us/RdTc

**Comment by andreas-eberle** [comment]
Can you try updating your graphics drivers to the newest version?

**Comment by ckorn** [comment]
Just upgraded to 364.12.

Now it does not crash any more but I only get a grey screen:
http://pix.toile-libre.org/upload/original/1460146448.png

Output is:
```
Found default command line options file. Starting to read.
Default command line options file successfully read.
Options loaded: {}
Could not find/load graphic file 01
Could not find/load graphic file 11
java.lang.NullPointerException
	at jsettlers.graphics.map.draw.ImagePreloadTask.run(ImagePreloadTask.java:121)
	at java.lang.Thread.run(Thread.java:745)
Could not find/load graphic file 12
java.lang.NullPointerException
	at jsettlers.graphics.map.draw.ImagePreloadTask.run(ImagePreloadTask.java:189)
	at java.lang.Thread.run(Thread.java:745)
Could not find/load graphic file 03
Could not find/load graphic file 03
Warning: Could not load LocaleSuffix [_de_DE]. Falling back to next file.
Loading Sequence number 17
Loading Sequence number 18
libEGL warning: DRI2: failed to authenticate
libEGL warning: DRI2: failed to authenticate
libEGL warning: DRI2: failed to authenticate
```

**Comment by andreas-eberle** [comment]
This seems to be errors in JOGL which isn't maintained any longer. That's why we want to switch to another graphics library. I'm closing this in favor of #585.
